src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/eventDetailsBuilder.hpp

```diff
@@ -229,6 +231,18 @@ class TEventDetailsBuilder final : public AbstractHandler<std::shared_ptr<TScanC
                 ecsData["vulnerability"]["published_at"] = returnData.data->datePublished()->str();
                 ecsData["vulnerability"]["reference"] = returnData.data->reference()->str();
                 ecsData["vulnerability"]["scanner"]["vendor"] = "Wazuh";
+                try
+                {
+                    ecsData["vulnerability"]["scanner"]["source"] = TGlobalData::instance()
```

Comment:
remove the try-catch, we only need to check if data->m_vulnerabilitySource.first exist in the global map.

adp_descriptions and adp key's will exist in all oportunities, and if these values dont exists is a scanner break condition.